site/guide/_manage-roles.qmd

```diff
@@ -0,0 +1,21 @@
+1. Under User Permissions, select **Roles**.
+
+1. Select the role for which you want to manage.
+
+1. Select **Edit Permissions** to edit its details. Within the role's details, you can:
+   - View the current permissions assigned to the role.
+   - Add new permissions by specifying the actions or resources the role should have access to.
+   - Remove permissions that are no longer necessary for the role.
+   - Save the updated permissions configuration for the role.
+
+1. Select **Users** to manage users assigned to the role.
```

Comment:
Pretty sure we'd like some step-by-step here, as this is the managing roles article. ;)